zebra-chain/src/lib.rs

```diff
@@ -42,6 +42,7 @@ pub mod value_balance;
 pub mod work;
 
 pub use error::Error;
+pub use zcash_primitives;
```

Comment:
I think it might be good to add a comment with the reason of adding this here:

```suggestion

// Re-export librustzcash crates so consumers use Zebra’s versions directly,
// avoiding duplicate builds or version mismatches.
pub use zcash_primitives;
```

I have the impression that `zcash_primitives` will not be the only create we will re-export here that is why the comment added says "Re-export librustzcash crateS ...".

Reply:
True, but I think that `zcash_primitives` is sufficient for the discussion.

It's a Proof Of Concept, if we like it, then yeah..  I assume we'll use the pattern everywhere we can.

I also wasn't sure which zebra crate would be the best choice, for Zaino..   or more generally?